scripts.v3/migrate.js

```diff
@@ -117,9 +131,13 @@ async function migrate() {
         const destIimporterExporter = new ImporterExporter(yargs.destSubscriptionId, yargs.destResourceGroupName, yargs.destServiceName, yargs.destTenantid, yargs.destServiceprincipal, yargs.destSecret);
         await destIimporterExporter.cleanup();
         await destIimporterExporter.import();
+
+        if (yargs.existingEnvUrls != "" && yargs.destEnvUrls != "") {
```

Comment:
@SSanjeevi, I still think it should be part of separate a script file (and invoked separately) because it is uncommon feature and no customers ever expressed intent to do something like this. We can't accept project-specific changes in commonly used scripts, otherwise we end up with unnecessarily bloated `migrate.js`. Hope that makes sense.

Reply:
This PR is a good example for the pattern: #1405

Reply:
Done as per your request added script file updatecontenturl.js file